kolena/_experimental/quality_standard.py

```diff
@@ -87,17 +295,38 @@ def download_quality_standard_result(
     :param metric_groups: The names of the metric groups to include in the result.
     :param intersect_results: If True, only include datapoint that are common to all models in the metrics calculation.
     Note all metric groups are included when this value is `None`.
+    :param confidence_level: The confidence score used to calculate the Margin of Error, representing the probability of
+     capturing the true population parameter within the calculated MOE, we recommend setting between 0.9 - 0.99.
+     If this is specified performance delta column will be shown, metric valued will be classified as
+      improved, regressed, similar or unknown according to the MOE.
+    :param reference_model: The name of the model to use as a reference for the Margin of Error calculation,
+    :param reference_eval_config: The evaluation configuration to use in conjunction with the reference model,
+     if unspecified the first model in the models list will be used
+     as a reference for the Margin of Error, for None eval config, pass in 'null' instead of None. If this is None,
+     the first eval config will be used.
     :return: A Dataframe containing the quality standard result.
     """
+    if reference_model and reference_model not in models:
+        raise IncorrectUsageError("reference model not in models")
```

Comment:
We can improve the message here:
```suggestion
        raise IncorrectUsageError(f"The specified reference model '{reference_model}' was not one of the provided models")
```